### PR TITLE
Task done

### PR DESCRIPTION
**Pull request by skilaren**

### CODE REVIEW

File: KazanExpressTestTask/settings.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+WSGI_APPLICATION = 'KazanExpressTestTask.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'kazanxtestdb',
```
Comment (kevinkhanda) [inline KazanExpressTestTask/settings.py:90]: What do you think about keeping secrets here?

Comment (skilaren) [reply KazanExpressTestTask/settings.py:90]: It has to be in .cnf file where DB settings will be stored

Comment (skilaren) [reply KazanExpressTestTask/settings.py:90]: Or in environment variables where Django is going to run

Comment (kevinkhanda) [reply KazanExpressTestTask/settings.py:90]: Ok

**Comment by kevinkhanda** [comment]
What about docker compose file? You mentioned it in README, but it was not provided. I know that this is your own preference to add it but anyway

File: READMEdone.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Setup 
+To setup database - there is the psql.yml file for docker-compose within psql.env for environement variables
+
+All necessary libraries are in 'requirements.txt'
```
Comment (kevinkhanda) [inline READMEdone.md:4]: There is no such file provided in repository

Comment (skilaren) [reply READMEdone.md:4]: It is ‘psql.yml’

Comment (skilaren) [reply READMEdone.md:4]: In the root of project

Comment (kevinkhanda) [reply READMEdone.md:4]: I mean `requirements.txt`

Comment (skilaren) [reply READMEdone.md:4]: I see now, accidentally miss it. Added

Comment (kevinkhanda) [reply READMEdone.md:4]: Tnx

**Pull request closed without merging**